config/settings/production.py

```diff
@@ -1,5 +1,3 @@
-from ._base import *  # noqa
```

Comment:
This breaks our settings as we rely on the defaults from _base and we only override prod specific settings